Competition2018/src/main/java/competition/subsystems/elevator/commands/MoveToMinHeightCommand.java

```diff
@@ -28,19 +24,21 @@ public MoveToMinHeightCommand(ElevatorSubsystem elevator, PIDFactory pf) {
     @Override
     public void initialize() {
         log.info("Initializing");
+        if (!elevator.isCalibrated()) {
+            log.warn("ELEVATOR UNCALIBRATED - THIS COMMAND WILL NOT DO ANYTHING!");
+        }
     }
 
     @Override
     public void execute() {
-        double power = pid.calculate(min, elevator.getCurrentHeight());
-
-        elevator.setPower(power);
-
+        if (elevator.isCalibrated()) {
+            double power = pid.calculate(elevator.getMinHeight(), elevator.getCurrentHeightInInches());
```

Comment:
See above; we should only have one copy of all this logic.

Reply:
^^

Reply:
Agreed, these can be removed now.